### ISSUE TITLE
return@ should be allowed for ReturnFromFinally

### ISSUE THREAD

**Issue by Mygod** [arturbosch/detekt#1467]
## Expected Behavior

A very realistic (but simplified) example code using Kotlin coroutines:

```kotlin
try {
    // ...
} finally {
    withContext(NonCancellable) {
        return@withContext
    }
}
```

No warnings should raise.

## Observed Behavior

`ReturnFromFinally`

## Steps to Reproduce

Run detekt.

## Your Environment

* Version of detekt used: 1.0.0.RC9.2
* Version of Gradle used (if applicable): 5.2
* Operating System and version: Codacy
* Link to your project (if it's a public repository):

**Comment by arturbosch** [comment]
I think a property option should be considered here.
